linter/eslint.config.js

```diff
@@ -0,0 +1,97 @@
+import js from '@eslint/js';
+import ts from '@typescript-eslint/eslint-plugin';
+import tsParser from '@typescript-eslint/parser';
+import eslint from 'eslint';
+import esImport from 'eslint-plugin-import';
+import unusedImports from 'eslint-plugin-unused-imports';
+import globals from 'globals';
+
+export default [
+  eslint.configs.recommended,
+  js.configs.recommended,
+  {
+    ignores: ['**/eslint.config.js'],
+  },
+  {
+    plugins: {
+      '@typescript-eslint': ts,
+      'unused-imports': unusedImports,
+      import: esImport,
+    },
+
+    languageOptions: {
+      parser: tsParser,
+
+      parserOptions: {
+        project: 'tsconfig.json',
+        // tsconfigRootDir: __dirname,
+        ecmaVersion: 'latest',
+      },
+
+      globals: {
+        ...globals.node,
+        ...globals.browser,
+        ...globals.jest,
+      },
+    },
+
+    rules: {
+      ...eslint.configs.recommended.rules,
+      ...ts.configs.recommended.rules,
```

Comment:
Brauct es das nochmal, wenn in den Zeilen 10 und 11 schon die Config hinterlegt ist, oder überschreiben sich die Elemente im Array das per Default exportiert wird?